templates/endpoints/README.md

```diff
@@ -151,6 +153,60 @@ Endpoints allows you to easily add new models by adding a single configuration f
 To learn more about how to customize or add new models, 
 see the [Model Registry](models/README.md).
 
+# Serving LoRA Models
+
+`serve_lora.yaml` and `query_lora.py` are provided for you in this template.
+- If you just want to deploy static LoRA models, you can list them in `multiplex_lora_adapters` config in the `serve_lora.yaml` file.
+- If you want to dynamically loading LoRA models to a running service, you can use `dynamic_lora_loading_path` config in the `serve_lora.yaml` file.
+- Both can be set at the same time and complement to each other.
+
+Make sure you replace `dynamic_lora_loading_path` and `HUGGING_FACE_HUB_TOKEN` config in `serve_lora.yaml` with your own values. And place the LoRA checkpoint in the `dynamic_lora_loading_path` bucket.
+Also, if you are loading static LoRA models, replace `lora_mirror_config.bucket_uri` in `lora_config.yaml` with your own bucket uri.
+
+To deploy the LoRA models, run:
+```shell
+serve run serve_lora.yaml
+```
+
+This will take up to a minute or so to load depending on the model size given the required worker type is already up.
+For dynamic LoRA models, make sure to have your LoRA checkpoint stored in `{base_path}/{base_model_id}:{suffix}:{id}` and change the model id accordingly in `query_lora.py`.
+
+To query the LoRA model, run:
+```shell
+python query_lora.py
+
+# Example output:
+# {
+#     "id": "meta-llama/Llama-2-7b-chat-hf:lora-model:1234-472e56b56039273c260e783a80950816",
+#     "object": "text_completion",
+#     "created": 1699563681,
+#     "model": "meta-llama/Llama-2-7b-chat-hf:lora-model:1234",
+#     "choices": [
+#         {
+#             "message": {
+#                 "role": "assistant",
+#                 "content": " Sure, I can do that! Based on the target sentence you provided, I will construct the underlying meaning representation of the input sentence as a single function with attributes and attribute values.\n\nThe function I have constructed is:\n\n['inform', 'available_on_steam'] [1] [developer] [Slightly Mad Studios] [/]  \n\nThe attributes are:\n\n[1] [release_year] [2012]\n[developer] [Slightly Mad Studios]"
+#             },
+#             "index": 0,
+#             "finish_reason": "stop"
+#         }
+#     ],
+#     "usage": {
+#         "prompt_tokens": 285,
+#         "completion_tokens": 110,
+#         "total_tokens": 395
+#     }
+# }
+```
+
+A few tips for serving LoRA models:
+1. LoRA base models should be passed in the serve config file `serve_lora.yaml` in the `multiplex_models` config.
```

Comment:
Everything is numbered 1 in this list

Reply:
Yep, this is a trick for writing numbered list in markdown. It will automatically increment the number when it's rendered so if we insert any thing in between, we don't need to change the number for all :)  
<img width="1399" alt="image" src="https://github.com/anyscale/templates/assets/7553988/ab1e5d53-e741-4e5d-900f-eb9cab538c96">